test/ui/cypress/code/knative/serving/showcase.js

```diff
@@ -102,6 +101,21 @@ class ShowcaseKservice {
   }
 
   doRemoveApp() {
+    const env = new Environment()
+    const rng = env.random().next()
+    const self = this
+    const ways = [
+      () => { return self.removeAppViaKubectl() },
+      // FIXME: This do not work on OCP 4.10+ See: https://issues.redhat.com/browse/OCPBUGSM-41912
+      // () => { return self.removeAppViaUI() },
+    ]
+    const idx = Math.floor(rng * ways.length)
```

Comment:
I don't get this one. Are you randomly choosing a way to remove the app? Why?

Reply:
The idea is to try using UI. If we been using only the kubectl we wouldn't detect this regression of ODC.

Reply:
Maybe you're right.

Reply:
Yeah we should go back to the ui deletion asap when it is fixed.

Reply:
This should be fixed already in 4.10.5: https://bugzilla.redhat.com/show_bug.cgi?id=2062146

I'll confirm that.

Reply:
Confirmed it worked via UI.